web/client/epics/wfsquery.js

```diff
@@ -177,14 +177,15 @@ export const wfsQueryEpic = (action$, store) =>
             const searchUrl = ConfigUtils.filterUrlParams(action.searchUrl, authkeyParamNameSelector(store.getState()));
             // getSelected Layer and merge layerFilter and cql_filter in params  with action filter
             const layer = selectedLayerSelector(store.getState());
+            const useLayerFilter = useLayerFilterSelector(store.getState());
 
-            const {layerFilter, params} = layer ?? {};
+            const {layerFilter, params} = (layer) ?? {};
```

Comment:
curiosity why this? to handle null or undefined?

Reply:
```suggestion
            const {layerFilter, params} = layer ?? {};
```
It comes from a previous version, where the layer was a more complex expression, so it needed the parenthesis. Not needed anymore now.